cypress/e2e/notebooks.cy.ts

```diff
@@ -53,4 +60,45 @@ describe('Notebooks', () => {
         cy.get('.ph-recording.NotebookNode').should('be.visible')
         cy.get('.NotebookRecordingTimestamp').should('contain.text', '0:00')
     })
+
+    it('Can add a number list', () => {
+        cy.get('li').contains('Notebooks').should('exist').click()
+        cy.get('[data-attr="new-notebook"]').click()
+        // we don't actually get a new notebook because the API is mocked
+        // so, press enter twice to "exit" the timestamp block we start in
```

Comment:
I believe we should also be able to press `esc` to remove the insertion suggestion if that's what this is trying to achieve